src/sql/workbench/services/objectExplorer/browser/dragAndDropController.ts

```diff
@@ -109,14 +109,21 @@ export class ServerTreeDragAndDrop implements IDragAndDrop {
 	}
 
 	private getProviderNameFromElement(element: TreeNode): string | undefined {
-
 		if (element.connection) {
 			return element.connection.providerName;
 		}
 
 		return this.getProviderNameFromElement(element.parent);
 	}
 
+	private getFinalStringForKusto(nodeTypeId: string, escapedName: string) {
+		if (nodeTypeId === 'Table' && escapedName.indexOf(' ') > 0) {
+			return `[@"${escapedName}"]`;
```

Comment:
I assume you need to escape " in the string as well with this, right?

Although I'm confused why this is necessary - the point of the square brackets is that everything inside is considered an object identifier so spaces should be included in that. If that isn't working then what's the point of the square brackets here?